mlflow/genai/__init__.py

```diff
@@ -1,4 +1,52 @@
+import warnings
+
 from mlflow.genai.evaluation import evaluate, to_predict_fn
 from mlflow.genai.scorers import Scorer, scorer
 
-__all__ = ["evaluate", "to_predict_fn", "Scorer", "scorer"]
+try:
+    with warnings.catch_warnings():
+        # Ignore warnings from the mlflow.genai.datasets module
+        warnings.filterwarnings("ignore", message="The `databricks-agents` package is required")
+        from mlflow.genai.datasets import EvaluationDataset
+
+    from mlflow.genai.labeling import (
+        Agent,
+        LabelingSession,
+        ReviewApp,
+        get_review_app,
+        create_labeling_session,
+        get_labeling_sessions,
+        get_labeling_session,
+        delete_labeling_session,
+    )
+except ImportError:
+    # Silently fail if the databricks-agents package is not installed
+    pass
+
+from mlflow.genai.datasets import (
+    create_dataset,
+    delete_dataset,
+    get_dataset,
+)
+
+
+__all__ = [
+    "evaluate",
+    "to_predict_fn",
+    "Scorer",
+    "scorer",
+    # Labeling
+    "Agent",
+    "LabelingSession",
+    "ReviewApp",
+    "get_review_app",
+    "create_labeling_session",
```

Comment:
* import fails when `create_labeling_session` is unavailable. Can we append it (and other classes/funcions) to `__all__` only when they are importable?

```
from mlflow.genai import *
```

Reply:
Fixed! not sure if I can add a unit test since wildcards are not allowed in functions.